### PR TITLE
[Docs Site] Lint sh codeblocks

### PR DESCRIPTION
**Pull request by KianNH**
Adds a Vale lint to spot invalid `sh` codeblocks (if they don't contain a `$ ` then the entire block is unselectable/uncopyable).

### CODE REVIEW

File: content/cloudflare-one/connections/connect-networks/do-more-with-tunnels/migrate-legacy-tunnels.md [hunk 1]
```diff
@@ -36,21 +36,21 @@ To migrate your legacy tunnels to the named tunnels architecture:
 3. [Create a tunnel](/cloudflare-one/connections/connect-networks/get-started/create-local-tunnel/#3-create-a-tunnel-and-give-it-a-name).
 
     ```sh
-    cloudflared tunnel create <TUNNEL-NAME>
+    $ loudflared tunnel create <TUNNEL-NAME>
```
Comment (kodster28) [inline content/cloudflare-one/connections/connect-networks/do-more-with-tunnels/migrate-legacy-tunnels.md]:
```suggestion
    $ cloudflared tunnel create <TUNNEL-NAME>
```

File: .github/styles/cloudflare/ShellCodeBlocks.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+# Warning: cloudflare.ShellCodeBlocks
+#
+# Checks that all `sh` codeblocks contain at least one `$`, to indicate a command.
+#
+# For a list of all options, see https://vale.sh/docs/topics/styles/
+extends: script
+message: "**Warning**: `sh` codeblocks must include a `$` character at the beginning of a command, otherwise it can't be copied. If this is a multi-line codeblock, please use the `bash` language. For more information, refer to [Code block guidelines](https://developers.cloudflare.com/style-guide/formatting/code-block-guidelines/)"
```
[the remark this answers is not in the document]
Comment (KianNH) [reply .github/styles/cloudflare/ShellCodeBlocks.yml:8]: This is what I get for adding Vale lints to YAML - will follow-up to only lint `data` directory.

**Review by kodster28 — APPROVED** [review]

**Comment by GregBrimble** [comment]
Ideally, what we should be doing is omitting the `$` and injecting that in with a CSS `sh-blocks::before { content: '$'; }`.

This more clearly marks this as illustration rather than actual content. See more: https://accessibleweb.com/question-answer/how-is-css-pseudo-content-treated-by-screen-readers/

**Comment by KianNH** [comment]
> Ideally, what we should be doing is omitting the `$` and injecting that in with a CSS `sh-blocks::before { content: '$'; }`.
> 
> This more clearly marks this as illustration rather than actual content. See more: https://accessibleweb.com/question-answer/how-is-css-pseudo-content-treated-by-screen-readers/

`[language="sh"] .CodeBlock--row-content::before` would work, but it'll need some regex tweaks in Prism since we rely on `$` to identify prompt (before & including `$`) and command (after `$`). Probably a good time to move in-line with the official [`shell-session` grammar](https://github.com/PrismJS/prism/blob/master/components/prism-shell-session.js) (made an internal ticket for this idea)

**Pull request merged**